lib/features/push_notification/presentation/controller/fcm_token_controller.dart

```diff
@@ -165,7 +177,14 @@ class FcmTokenController extends FcmBaseController {
   @override
   void handleSuccessViewState(Success success) {
     log('FcmTokenController::_handleSuccessViewState(): $success');
-    if (success is GetFirebaseRegistrationByDeviceIdSuccess &&
+    if (success is GetAuthenticatedAccountSuccess &&
+        success.account.accountId != null &&
+        success.account.userName != null) {
+      _getStoredFirebaseRegistrationFromCache(
+        success.account.accountId!,
+        success.account.userName!
+      );
```

Comment:
How about if we combine it in one interactor `getActiveAccount + `_getStoredFirebaseRegistrationFromCache`?

Reply:
We shouldn't do that, because we won't call it when `FCM` is disabled.